packages/application-shell/src/components/project-switcher/project-switcher.js

```diff
@@ -135,12 +135,14 @@ export class ProjectSwitcher extends React.PureComponent {
         className={classnames(styles['item-text-main'], {
           [styles['item-text-disabled']]:
             (project.suspension && project.suspension.isActive) ||
-            (project.expiry && project.expiry.isActive),
+            (project.expiry &&
+              (project.expiry.isActive || project.expiry.daysLeft < 0)),
         })}
       >
         {project.name}
         {((project.suspension && project.suspension.isActive) ||
-          (project.expiry && project.expiry.isActive)) && (
+          (project.expiry &&
+            (project.expiry.isActive || project.expiry.daysLeft < 0))) && (
```

Comment:
Given this spreads so much it might have been a good idea to have `expiry.hasPassed` but I don't mind. Just a bit odd with the negative number.